charts/pomerium/templates/_helpers.tpl

```diff
@@ -638,11 +638,20 @@ true
 Return the appropriate apiVersion for ingress.
 */}}
 {{- define "ingress.apiVersion" -}}
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" -}}
-{{- print "networking.k8s.io/v1beta1" -}}
-{{- else -}}
-{{- print "extensions/v1beta1" -}}
+  {{- if and (.Capabilities.APIVersions.Has "networking.k8s.io/v1") (semverCompare ">= 1.19-0" .Capabilities.KubeVersion.Version) -}}
```

Comment:
For my own information, why is the semver compare needed?  I'd expect just the v1 api being present is enough but I'm guessing this is here for a reason.

Reply:
Fixed